src/lucky_record/errors.cr

```diff
@@ -12,6 +14,13 @@ module LuckyRecord
     end
   end
 
+  # Raised when Lucky Record cannot find a record by a query_method like #first or #last
+  class RecordNotFoundError < LuckyRecordError
+    def initialize(@model : Symbol, @id : Symbol)
+      super "Could not find #{id} #{LuckyInflector::Inflector.singularize(model)}"
```

Comment:
Maybe this could be simplified to: “could not find first record in #{model}”. You could then pass in “first” so later you could also pass in “last” or whatever else we decide to add later.